adapters/postgres/postgres.go

```diff
@@ -84,7 +86,9 @@ func (s *Stmt) Prepare(db *sqlx.DB, tx *sql.Tx, SQL string) (statement *sql.Stmt
 
 // Load postgres
 func Load() {
-	config.PrestConf.Adapter = &Postgres{}
+	config.PrestConf.Adapter = &Postgres{
+		enforcer: e,
+	}
 	db, err := connection.Get()
```

Comment:
Additionally the `casbin.Enforcer` could just be in the `config.PrestConf` if that's preferable.